src/client/helpers/entity.js

```diff
@@ -168,6 +168,18 @@ export function getEntityDisambiguation(entity) {
 	return null;
 }
 
+export function getEntityAlias(entity) {
+	const aliases = entity.aliasSet.aliases
+		.map(item => item.name)
+		.filter(item => item !== entity.defaultAlias.name)
+		.join(', ');
+	if (entity.aliasSet.aliases.length > 1) {
+		return <h5>{aliases}</h5>;
```

Comment:
I tried it out on my computer, and I find the aliases a tad small. Could you try an h4?